current_fall2025/scripts/load_bpl_vector.py

```
@@ -0,0 +1,114 @@
+#!/usr/bin/env python3
```

Ruff (0.14.0)

Shebang is present but file is not executable

(EXE001)